Pod/Classes/WPMediaCollectionViewCell.m

```diff
@@ -308,6 +308,8 @@ - (void)setPosition:(NSInteger)position
 {
     _position = position;
     if (position != NSNotFound) {
+        CGFloat fontSize = position < 100 ? 13.0 : 9.0;
+        _positionLabel.font = [UIFont systemFontOfSize:fontSize weight:UIFontWeightMedium];
```

Comment:
Could we pull the font sizes and weight out into constants so we only need to change them in one place? (we're setting 13 and medium in 2 different places)

I blame @jleandroperez for my pickiness about these things, sorry :)